workflow-bot-app/src/main/resources/application.yaml

```diff
@@ -39,7 +39,7 @@ spring:
     username: sa
     password:
     driver-class-name: org.h2.Driver
-    url: jdbc:h2:mem:process_engine;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE
+    url: jdbc:h2:mem:process_engine;DB_CLOSE_DELAY=-1;DB_CLOSE_ON_EXIT=FALSE;OLD_INFORMATION_SCHEMA=TRUE
```

Comment:
As discussed, please remove this new flag from the connection string.
Otherwise, LGTM